include/mbedtls/config.h

```diff
@@ -1755,6 +1755,8 @@
  */
 #define MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL
 
+#define MBEDTLS_RSA_PSS_RSAE_SHA256
```

Comment:
I don't think we need a new configuration option here, but we should rather reuse the existing configuration options for RSA from upstream Mbed TLS.

Reply:
I removed the new configuration option, and uses `MBEDTLS_X509_RSASSA_PSS_SUPPORT` from upstream, let me know if you mean different existing RSA configuration.